podvm/addons/.gitignore

```diff
@@ -0,0 +1 @@
+.enable
```

Comment:
If this is in .gitignore you cannot build images with addons on a CI.

Reply:
I see..

Reply:
yes, but then you have to alter the workflow. often workflows and OIDC identities are tied to a specific branch (main), so you'll run the workflow on main, but specify a "ref" that will be used in the `checkout` step to build a specific revision.